### DISCUSSION TITLE
2017-06-15 - Release Day

### DISCUSSION THREAD

**Discussion by drdanz** [robotology/community#228] [Announcements]
Tomorrow, 2017-06-15, we are going to release several robotology "core" packages (RTF, YCM, YARP,  and iCub-*).
We are going to merge `devel` into `master` for all the repositories, therefore there could be issues if you are on the `master` branch and you update the repositories while the some are merged and other are not.
Therefore we suggest not to pull the master branch until the releases are done (announcements will follow).

@robotology/all-robotology-developers

**Comment by traversaro** [comment]
For anyone reading: the release process is still ongoing at the moment. 
 cc @gabrielenava @S-Dafarra

**Comment by francesco-romano** [comment]
I would reopen this issue until we finish the process @pattacini

**Comment by drdanz** [comment]
Everything was released.
Closing this issue
